src/index.js

```diff
@@ -27,6 +27,20 @@ const COLOR_SCALES = Object.freeze({
 });
 /* eslint-enable max-len */
 
+function setDimensionLimit(handler) {
+  var dims = handler.getDimensions();
+  if(dims && dims.length) {
```

Comment:
Unnecessary check for dims.length, .forEach works on zero-length arrays.
```suggestion
  if(dims) {
```

Reply:
But I would just turn the condition into an early exit instead,
`
if(!dims) return;
`